source/includes/materials/_32-git-notification.md.erb

```diff
@@ -54,3 +54,24 @@ curl 'https://ci.example.com/go/api/material/notify/git' \
     -X POST \
     -d "repository_url=git://git.example.com/git/funky-widgets.git"
 ```
+
+
+
+### Hosting your repository on GitHub [Enterprise]?
+
+__Available since v17.6__ support for GitHub organisation webhook notifications.
```

Comment:
Maybe use the `available_since` tag? Like [this](https://github.com/stephen-murby/api.go.cd/blob/7c43d6f5232bdc407a53bcfce27986c1a9529baf/source/includes/materials/_32-git-notification.md.erb#L24).